order-a-coffee.json

```diff
@@ -1,15 +1,6 @@
 {
   "title": "order-a-coffee",
   "steps": [
-    {
```

Comment:
@IdosApplitools This file is exported from the Chrome's Recorder, we should not change anything here. We should make our code work for that.

Reply:
@jecfish Using `visualGridOptions` and IEv2 we are now able to see a successful execution for IE11 browser as well.
The cause for that was the blank page that we see from the `setViewport` step